variables.tf

```diff
@@ -30,24 +31,58 @@ variable "resource_labels" {
 // ----------------------------------------------------------------------------
 // cluster configuration
 // ----------------------------------------------------------------------------
+variable "master_authorized_networks" {
```

Comment:
can we document this field in the readme?

Reply:
added a readme for this